src/hooks/member.ts

```diff
@@ -89,13 +83,18 @@ export default (queryConfig: QueryClientConfig) => {
       const queryClient = useQueryClient();
       let shouldFetch = true;
       if (id) {
+        // TODO: this casting is totally wrong, but allows to work for current member
+        // to be fixed
         shouldFetch =
-          queryClient.getQueryData<MemberRecord>(buildMemberKey(id))?.extra
-            ?.hasAvatar ?? true;
+          (
+            queryClient.getQueryData<Member>(
+              buildMemberKey(id),
+            ) as CompleteMember
+          )?.extra?.hasAvatar ?? true;
```

Comment:
Let's see how we can handle this. Maybe we always make the request but the backend checks if the member has an avatar or not and returns a 200 OK in both cases but without the url if the member has no avatar as this might be expected. TBD.